From 02ad73da55ee47850ef30c3b4d97e60e1eea9b6b Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Mon, 2 Oct 2023 16:18:22 +0200
Subject: [PATCH] Update com.webos.service.notification.pem.json.in: Fix
 listApps call

Fixes:

Oct 02 15:05:02 qemux86-64 sam[544]: [] [pmlog] <default-lib> LS_REQUIRES_SECURITY {"SERVICE":"com.webos.notification","CATEGORY":"/","METHOD":"listApps"} Service security groups don't allow method call.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 files/sysbus/com.webos.notification.perm.json.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.notification.perm.json.in b/files/sysbus/com.webos.notification.perm.json.in
index 920c439..8df32ec 100644
--- a/files/sysbus/com.webos.notification.perm.json.in
+++ b/files/sysbus/com.webos.notification.perm.json.in
@@ -1,5 +1,5 @@
 {
     "com.webos.notification": [
-         "database.operation","applicationmanager.query","time.query","settings.query","tv.settings"
+         "database.operation","applicationmanager.query","time.query","settings.query","tv.settings","application.operation"
     ]
 }
